### PR TITLE
ci: Setup dependabot for github-actions

### LINKED ISSUE #1502: Setup dependabot for github actions
## Summary

Automatically update third-party github actions with dependabot.

## Motivation

Some third-party github actions may become outdated and it is convenient to receive notiffications (or maybe even PRs) about it.

### PR DESCRIPTION
**Pull request by godexsoft**
Fixes #1502

### CODE REVIEW

**Review by cindyyan317 — DISMISSED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Etc/GMT"
+    reviewers:
+      - "cindyyan317"
+      - "godexsoft"
+      - "keznetsss"
```
Comment (kuznetsss) [inline .github/dependabot.yml]: `kuznetsss`

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    commit-message:
+      prefix: "[CI] "
```
Comment (kuznetsss) [inline .github/dependabot.yml:15]: It should follow conventional commits, so `ci: ` probably.

File: .github/dependabot.yml (hunk 1 above)
Comment (godexsoft) [reply .github/dependabot.yml]: How do i mess that up 🔢 🤷

File: .github/dependabot.yml (hunk 2 above)
Comment (godexsoft) [reply .github/dependabot.yml:15]: Notice this is not the title, it's the commit message - this is the same as we have for our clang-tidy commit message. The title of the PR should already be something like `chore:` or `ci:` and we can update it to what we like so dependabot can remember it for later PRs it opens.

**Review by kuznetsss — APPROVED** [review]

**Pull request merged**